ui/apps/platform/src/Containers/NetworkGraph/deployment/deploymentFlows.css

```diff
@@ -0,0 +1,3 @@
+.pf-c-table__toggle {
```

Comment:
I only needed to do this tweak so the toggle doesn't take up so much space with padding

Reply:
Open to suggestions on how to organize CSS files. I put it on the same level as the component that uses it and same naming structure